2022-07-17 04:05:35,713 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-07-17 04:05:35,718 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\ridwn\AppData\Local\Programs\Python\Python39\lib\site-packages\instabot\bot\bot.py
2022-07-17 04:05:35,719 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-07-17 04:05:35,905 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-07-17 04:05:35,905 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-07-17 04:05:35,905 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-07-17 04:05:35,912 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
